### ISSUE TITLE
[TS-171] pylint: Module 'http.server' has no 'ThreadingHTTPServer' member

### ISSUE THREAD

**Issue by mirabilos** [codacy/codacy-analysis-cli-action#98]
Erm, it does. That’s what I run.

**Comment by DMarinhoCodacy** [comment]
Hello @mirabilos 

Thanks for reporting this.

Are you still experiencing the same behavior?

Kind regards,
David Marinho

**Comment by mirabilos** [comment]
Yes, it shows still up <https://github.com/tarent/jensjs/security/code-scanning/671>, together with even more error messages that are not even errors

**Comment by DMarinhoCodacy** [comment]
Hello @mirabilos,

I've made some tests with http.server and ThreadingHTTPServer is not returning any error since we bumped Pylint to a recent version (not the latest one yet but we'll do that very soon). But, either way, you're completely right. [According to the docs, ThreadingHTTPServer is part of http.server since Python3.7](https://docs.python.org/3/library/http.server.html).

Can you take some of your time and test if you still have this error?

Thanks in advance for your help.

Kind regards,
David Marinho

**Comment by mirabilos** [comment]
Ah, unfortunately it seems I *cannot* retest this, as it fails to run nowadays:

```
Error: Code Scanning could not process the submitted SARIF file:
rejecting SARIF, as there are more runs than allowed (21 > 20)
```

I think this is because too many test utilities run, or something.

**Comment by mirabilos** [comment]
Ah hmm. <https://docs.codacy.com/repositories-configure/codacy-configuration-file/#legacy-pylint-19> says to disable pylint and rely on pylintpython3.

Which is what I want, actually.

**Comment by mirabilos** [comment]
However, this…

```
$ cat .codacy.yml                                                               
---
engines:
  pylint:
    # pylintpython3 is used instead
    enabled: false
  tsqllint:
    # for Microsoft only
    enabled: false
```

… does not disable either ☹☹☹
